dependencies/direct/py.in

```diff
@@ -4,7 +4,9 @@ covdefaults
 coverage  # accessed directly from tox
 coverage-enable-subprocess
 hypothesis
+jinja2
 pytest
 pytest-cov
 pytest-mock
 pytest-xdist
+pyyaml
```

Comment:
This is an incorrect place for declaring this dependency. It's not test-only, so it mustn't be here. It's a runtime dependency, so we should have it in the core packaging metadata.